template/vtex/manifest.json

```diff
@@ -0,0 +1,18 @@
+{
+  "vendor": "CHANGE_ME",
```

Comment:
you can change the `CHANGE_ME` of this file by the variables that were defined there in the package.json

like this https://github.com/vtex/create-react-library/blob/master/template/vtex/package.json#L2